src/main/java/Deadline.java

```diff
@@ -0,0 +1,22 @@
+public class Deadline extends Task {
+    protected String completeBy;
+//    protected boolean isDone;
+
+
+    public Deadline(String description, String completeBy) {
+        super(description);
+        this.workType = 'D';
+        this.completeBy = completeBy;
+    }
+
+    public String getStatusIcon() {
+        return (isDone ? "\u2713" : "\u2718"); //return tick or X symbols
```

Comment:
Consider using an enum/constant for these unicode characters.
Most people wouldn't know the identity of these characters off the top of their heads.
Even though the IDE may render it for you, this would benefit the reviewers and you in the future if you have plan on using these characters more in the codebase